lib/tasks/model.rake

```diff
@@ -8,7 +8,7 @@
 
 desc 'Add column annotations to active records'
 task :annotate do
-  sh 'annotate -p before -e tests'
+  sh 'RAILS_ENV=test annotate -p before -e tests'
```

Comment:
Funktioniert die Annotierung im Test-ENV besser? Oder ist das direkt eine Vorbedingung, alle Dateien zu erwischen?

Reply:
Bei mir ist im development environment zumeist in Wagon aktiv, daher haben wir mit unter zusätzliche Spalten auf den Tabellen. Im test environment haben ich wenn ich core tests laufe nur die migrations aus dem core

Reply:
Ah, verstehe. Dann ist das erstmal besser so als ohne die Angabe.